api/HTMLScriptElement.json

```diff
@@ -692,6 +739,53 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "accepts_TrustedScriptUrl": {
+          "__compat": {
+            "description": "Can be set with a `TrustedScriptUrl`",
```

Comment:
Is this the right way to record this kind of feature change? Note that the value can now be set with a string or TrustedScriptURL, but if fetched it will be a string.

Reply:
This seems reasonable to me.

Reply:
```suggestion
        "accepts_TrustedScriptUrl": {
          "__compat": {
            "description": "Can be set with a `TrustedScriptURL` instance",
```